packages/language-server/tests/definitions.test.js

```diff
@@ -0,0 +1,137 @@
+/**
+ * @typedef {import('vscode-languageserver').ProtocolConnection} ProtocolConnection
+ */
+import assert from 'node:assert'
```

Comment:
```suggestion
import assert from 'node:assert/strict'
```

If possible (not on older Node I believe), I prefer using the strict assertions!

Reply:
Nice one!